examples/nirfmxbluetooth/txp-basic.py

```diff
@@ -297,22 +298,24 @@ def raise_if_error(response):
         f"Peak to Average Power Ratio Maximum (dB)        : {peak_to_average_power_ratio_maximum}"
     )
 
-    # Uncomment this block if these values are generated by your measurement:
-    # print(f"EDR GFSK Average Power Mean (dBm)               : {edrgfsk_average_power_mean}")
-    # print(f"EDR DPSK Average Power Mean (dBm)               : {edrdpsk_average_power_mean}")
-    # print(
-    #     f"EDR DPSK GFSK Average Power Ratio Mean (dB)     : {edR__dpsK__gfsk_average_power_ratio_mean}"
-    # )
+    if basic_rate_packets == False:
+      print(f"EDR GFSK Average Power Mean (dBm)               : {edrgfsk_average_power_mean}")
+      print(f"EDR DPSK Average Power Mean (dBm)               : {edrdpsk_average_power_mean}")
+      print(
+          f"EDR DPSK GFSK Average Power Ratio Mean (dB)     : {edR__dpsK__gfsk_average_power_ratio_mean}"
+      )
 
-    # Uncomment this block if these values are generated by your measurement:
-    # print("------------------LE CTE Reference Period Measurement------------------")
-    # print(
-    #     f"Average Power Mean (dBm)                                         : {reference_period_average_power_mean}"
-    # )
-    # print(
-    #     f"Peak Absolute Power Deviation Maximum (%)                        : {reference_period_peak_absolute_power_deviation_maximum}\n"
-    # )
+    # These results are invalid for basic rate packets.
+    if basic_rate_packets == False:
+      print("------------------LE CTE Reference Period Measurement------------------")
+      print(
+          f"Average Power Mean (dBm)                                         : {reference_period_average_power_mean}"
+      )
+      print(
+          f"Peak Absolute Power Deviation Maximum (%)                        : {reference_period_peak_absolute_power_deviation_maximum}\n"
+      )
 
+    # These results are invalid for basic rate packets.
```

Comment:
Looks like you accidentally copied this comment here, since the lines below aren't under a check like `if basic_rate_packets == False`